src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -733,6 +733,38 @@ namespace winrt::TerminalApp::implementation
         {
             _noMatchesText().Visibility(Visibility::Collapsed);
         }
+
+        if (_currentMode == CommandPaletteMode::CommandlineMode)
+        {
+            ParsedCommandLineText(L"");
+
+            const auto commandLine = _getTrimmedInput();
+            if (!commandLine.empty())
+            {
+                ExecuteCommandlineArgs args{ commandLine };
+                ::TerminalApp::AppCommandlineArgs appArgs;
+                appArgs.DisableHelpInExitMessage();
+
+                if (appArgs.ParseArgs(args) == 0)
+                {
+                    const auto& commands = appArgs.GetStartupActions();
+                    if (commands.size() > 0)
+                    {
+                        std::wstring commandDescription{ RS_(L"CommandPalette_ParsedCommandLine") + L":" };
```

Comment:
(asking the team) Not too familiar with how localization works, but should we move the `:` to the resw?

Reply:
@DHowett would know for sure, but my guess would be _yes, we need to move it to the resw_. I'm thinking the resource would need to be 

```
Executing command line will invoke the following commands:{0}
```

and then we `fmt::format` that with the list of generated names (joined with `\n\t`)

Reply:
I do agree that `:` should be there. My idea was not to include `{0}` so if we decide to put them in different Runs (e.g. make this caption red or green) we won't need new translation. WDYT?

Reply:
Yeah, I actually agree with that. We may want the actions to each be a separate run of text for whatever reason.